elasticsearch/load_cancerbiomarker.sh

```diff
@@ -0,0 +1 @@
+cat "$1" | elasticsearch_loader --es-host "http://localhost:9200" --index-settings-file "index_settings.json" --bulk-size 5000 --index cancerbiomarker --type cancerbiomarker --id-field id json --json-lines -
```

Comment:
@cmalangone `es >= 7` does not accept type anymore. Did you try loading into es7 with this script?

Reply:
It is still supported.